src/util/debug_module.js

```diff
@@ -379,7 +389,8 @@ class InternalDebugLogger {
     }
 
     log_internal(msg_info) {
-        if (syslog && config.LOG_TO_SYSLOG_ENABLED) {
+        if (syslog && config.LOG_TO_SYSLOG_ENABLED &&
+            this._levels[msg_info.level] < LOG_DEBUG_LEVEL) {
```

Comment:
@naveenpaul1 
On syslog condition shouldn't it be <= ? 
Asking because on stderr we skipped every msg above LOG_LEVEL, so we printed msgs if this._levels[msg_info.level] <= LOG_LEVEL

Reply:
@romayalon you are right, We need all the logs till `LOG_DEBUG_LEVEL` , Will update,